modules/learn/pages/clusters-and-availability/automatic-failover.adoc

```diff
@@ -8,39 +9,72 @@
 
 == Understanding Automatic Failover
 
+Important: The concept of auto-failover, and failover in general, applies to nodes in a single cluster, including nodes in a Server Group. There are no separate settings for Server Group failover or Server Group auto-failover. 
+If auto-failover settings are configured properly, and if the cluster services topology allows, auto-failover of all the nodes in a Server Group can occur. For example, if an event occurs so that all of the nodes in a Server Group were to become unavailable so that they were eligible for auto-failover, for auto-failover of all the nodes in the Server Group to occur, all of the auto-failover constraints listed in xref:learn:clusters-and-availability/automatic-failover.adoc[Auto-failover Constraints] must be met, including the majority quorum requirement: i.e., the remaining nodes must be able to form a majority quorum to be able to initiate an auto-failover. 
+
+See xref:learn:clusters-and-availability/groups.adoc[Server Group Awareness] for more information about Server Groups.
+
+Note that node failover in the Couchbase Server is in the context of a single cluster, and auto-failover only occurs in a single cluster.
+In the context of Cross Data Center Replication (XDCR), the failover refers to application failover to a different cluster. Application failovers are always determined and controlled by the user.
```

Comment:
Please remove the last sentence, as I do not think that it's necessary: 
"Application failovers are always determined and controlled by the user."